src/app/shared/services/layout.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LayoutService {
+  // Service allowing modifications of the layout
+
+  private leftMenuAndHeadersDisplayed = new BehaviorSubject<boolean>(true);
+  leftMenuAndHeadersDisplayed$ = this.leftMenuAndHeadersDisplayed.asObservable().pipe(delay(0));
+
+  private withTask = new BehaviorSubject<boolean>(false);
+  withTask$ = this.withTask.asObservable().pipe(delay(0));
+
+  toggleLeftMenuAndHeaders(shown?: boolean): void {
+    if (shown !== undefined) {
+      this.leftMenuAndHeadersDisplayed.next(shown);
+    } else {
+      this.leftMenuAndHeadersDisplayed.next(!this.leftMenuAndHeadersDisplayed.value);
```

Comment:
I see only usage with a given parameter so I does not look relevant case.

Reply:
Removed.